### PR TITLE
Update 0.8.0 code

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

### CODE REVIEW

**Comment by tpmccallum** [comment]
Thanks, @Ivan, will do.

**Comment by radu-matei** [comment]
`spin oci run` has been removed in v0.9.

`spin up --from-registry <reference>` is the command for running an app from the registry.

File: content/spin/spin-oci.md [hunk 1]
```diff
@@ -126,8 +126,7 @@ Lastly, let's run this Spin application.
  <!-- @selectiveCpy -->
 
  ```bash
- # make sure you've built your application with `spin build` prior
- spin up --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
+ spin oci run --from-registry ghcr.io/USERNAME/spin-react-fullstack:v1
```
Comment (radu-matei) [inline content/spin/spin-oci.md:129]: `spin oci run` has been removed in v0.9.

Comment (itowlson) [reply content/spin/spin-oci.md:129]: I think this is in an 0.8 tab though - that was my interpretation of lines 124 to here.  Am I misreading?

Comment (karthik2804) [reply content/spin/spin-oci.md:129]: @itowlson, you are reading it correctly.

**Pull request merged**